jekyll/_cci2/container-runner.adoc

```diff
@@ -95,6 +95,177 @@ agent:
           - name: "other"
 ```
 
+[#customizable-service-containers]
+=== Customizable service containers
+
+By default, link:https://circleci.com/docs/using-docker/#using-multiple-docker-images[service (or secondary) containers] inherit the same link:https://kubernetes.io/docs/tasks/configure-pod-container/[container configuration] as defined by the primary container. However, this behavior can be overridden using customizable service containers. This allows fine-tuned control over a service's resource usage on a per-image basis.
+
+[#customizable-service-containers-example]
+==== Example
+
+Consider the following container runner Helm values:
+
+[source,yaml]
+----
+agent:
+  serviceContainers:
+    exact:
+      - "cimg/redis:6":
+        resources:
+          requests:
+            cpu: "0.5"
+            memory: "200Mi"
+  resourceClasses:
+    your-namespace/your-resource-class:
+      serviceContainers:
+        exact:
+          - "cimg/postgres:16":
```

Comment:
These go into a go map, so we don't want the `-`

This applies to all the match types.

I forgot to update the confluence doc & helm example, will do both now

Reply:
Already fixed :)